### PR TITLE
feat: project group access list delete

### PR DESCRIPTION
**Pull request by IrakliJani**
### Description:

CleanShot 2024-01-11 at 12 48 59@2x

CleanShot 2024-01-11 at 12 48 54@2x

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/ProjectAccess/index.tsx [hunk 1]
```diff
@@ -20,7 +20,8 @@ const ProjectUserAccess: FC<ProjectUserAccessProps> = ({ projectUuid }) => {
     const [showProjectGroupAccessAdd, setShowProjectGroupAccessAdd] =
         useState(false);
 
-    const groupManagementEnabled = useFeatureFlagEnabled('group-management');
+    const groupManagementEnabled =
+        useFeatureFlagEnabled('group-management') || true;
```
Comment (magnew) [inline packages/frontend/src/components/ProjectAccess/index.tsx:24]: Won't this 'true' turn this on everywhere?

File: packages/frontend/src/features/projectGroupAccess/components/RemoveProjectGroupAccessModal.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                {group.name}"?
+            </Text>
+
+            <Group spacing="xs" position="right">
+                <Button variant="outline" onClick={onClose} color="dark">
+                    Cancel
+                </Button>
+
+                <Button color="red" onClick={onDelete}>
+                    Delete
```
Comment (magnew) [inline packages/frontend/src/features/projectGroupAccess/components/RemoveProjectGroupAccessModal.tsx:40]: This should probably say 'Revoke' to respond to the question asked.

**Review by magnew — COMMENTED** [review]
Looks good. I have a copy suggestion. And a question about the feature flag .

File: packages/frontend/src/components/ProjectAccess/index.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/ProjectAccess/index.tsx:24]: good catch, I was doing that because feature flags act weird 🤦

Comment (magnew) [reply packages/frontend/src/components/ProjectAccess/index.tsx]: Yea...I wonder whats up there.

**Review by magnew — APPROVED** [review]
👍

**Pull request merged**